switchapi/es2k/switch_pd_utils.c

```diff
@@ -194,6 +187,187 @@ tdi_status_t switch_pd_get_physical_port_id(switch_device_t device,
   return switch_pd_tdi_status_to_status(status);
 }
 
+tdi_status_t switch_pd_get_bridge_id(switch_device_t device,
+                                     uint8_t physical_port_id,
+                                     uint8_t* bridge_id) {
+  tdi_status_t status;
+
+  tdi_id_t field_id = 0;
+  tdi_id_t action_id = 0;
+  tdi_id_t data_field_id = 0;
+
+  tdi_dev_id_t dev_id = device;
+
+  tdi_flags_hdl* flags_hdl = NULL;
+  tdi_target_hdl* target_hdl = NULL;
+  const tdi_device_hdl* dev_hdl = NULL;
+  tdi_session_hdl* session = NULL;
+  const tdi_info_hdl* info_hdl = NULL;
+  tdi_table_key_hdl* key_hdl = NULL;
+  tdi_table_data_hdl* data_hdl = NULL;
+  const tdi_table_hdl* table_hdl = NULL;
+  const tdi_table_info_hdl* table_info_hdl = NULL;
+  uint32_t network_byte_order = 0;
+  uint64_t get_bridge_id = 0;
```

Comment:
bridge_id is uint8_t. Do we really need uint64_t here?

Reply:
tdi_data_field_get_value is taking uint64 as an argument which return the bridge value.
I can take it up as backlog and UT the changes to check if "tdi_data_field_get_value_ptr" can be used .